contracts/src/v0.8/automation/dev/v2_2/AutomationRegistryBase2_2.sol

```diff
@@ -307,16 +309,16 @@ abstract contract AutomationRegistryBase2_2 is ConfirmedOwner, ExecutionPreventi
 
   /// @dev Config + State storage struct which is on hot transmit path
   struct HotVars {
-    uint8 f; // maximum number of faulty oracles
-    uint32 paymentPremiumPPB; // premium percentage charged to user over tx cost
-    uint32 flatFeeMicroLink; // flat fee charged to user for every perform
-    uint24 stalenessSeconds; // Staleness tolerance for feeds
-    uint16 gasCeilingMultiplier; // multiplier on top of fast gas feed for upper bound
-    bool paused; // pause switch for all upkeeps in the registry
-    bool reentrancyGuard; // guard against reentrancy
-    uint96 totalPremium; // total historical payment to oracles for premium
-    uint32 latestEpoch; // latest epoch for which a report was transmitted
-    // 1 EVM word full
+    uint96 totalPremium; // ─────────╮ total historical payment to oracles for premium
+    uint32 paymentPremiumPPB; //     │ premium percentage charged to user over tx cost
+    uint32 flatFeeMicroLink; //      │ flat fee charged to user for every perform
+    uint32 latestEpoch; //           │ latest epoch for which a report was transmitted
+    uint24 stalenessSeconds; //      │ Staleness tolerance for feeds
+    uint16 gasCeilingMultiplier; //  │ multiplier on top of fast gas feed for upper bound
+    uint8 f; //                      │ maximum number of faulty oracles
+    bool paused; //                  │ pause switch for all upkeeps in the registry
+    bool reentrancyGuard; // ────────╯ guard against reentrancy
+    bool skipReorgProtection; //     if this registry should skip re-org protection mechanism
```

Comment:
idea for the future - we can totally combine these booleans into a single uint8 using a bitmap, then the struct would be 1 word again :)

Reply:
I created a ticket and assigned to myself: https://smartcontract-it.atlassian.net/browse/AUTO-8859